python-interface/src/MiniBotScripts/SwarmMaster.py

```diff
@@ -19,11 +20,102 @@ def run(bot):
     mediateThread = Thread(target=z.mediate)
     mediateThread.start()
     threads.append(mediateThread)
-    # commands for bot movement itself
+    
+    #echobot(bot,z)
+    colorbot(bot,z)
+    
+def colorbot(bot,z):
```

Comment:
how does the minibot main script know which of these to run? I thought we had required a run(bot) function.